gradio/components/multimodaltextbox.py

```diff
@@ -0,0 +1,267 @@
+"""gr.MultimodalTextbox() component."""
+
+from __future__ import annotations
+
+from typing import Callable, Literal, Any
+import warnings
+import tempfile
+
+from gradio import utils
+from gradio_client import utils as client_utils
+from gradio_client.documentation import document, set_documentation_group
+from gradio_client.serializing import StringSerializable
+from gradio_client.serializing import FileSerializable
+
+from gradio.components.base import (
+    FormComponent,
+    IOComponent,
+    _Keywords,
+)
+from gradio.events import (
+    Blurrable,
+    Changeable,
+    EventListenerMethod,
+    Inputable,
+    Selectable,
+    Submittable,
+    Clickable,
+    Uploadable,
+)
+
+set_documentation_group("component")
+
+
+@document()
+class MultimodalTextbox(
```

Comment:
I think the name `MultimodalTextbox` is a little unclear. What about `RichTextbox` as that seems to be used in other places as well: https://www.google.com/search?q=richtextbox&oq=richtextbox&aqs=chrome..69i57j0i67i650j0i20i263i512l2j0i512l3j69i60.1407j0j7&sourceid=chrome&ie=UTF-8